deployment/Single Host/metrics/makeInstallers.sh

```
@@ -0,0 +1,152 @@
+cat <<EOT >> "$buildDir/$debDir/DEBIAN/postrm"
+#!/bin/bash
+
+systemctl daemon-reload
+# Remove docker image
+if [[ "$(docker images -q oqm-metrics-otel_lgtm 2> /dev/null)" != "" ]]; then
+        docker rmi oqm-metrics-otel_lgtm
+        echo "Removed docker image."
+else
+        echo "Docker image was already gone."
+fi
+if [ $( docker ps -a | grep oqm-metrics-otel_lgtm | wc -l ) -gt 0 ]; then
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)